buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -434,7 +434,7 @@ class ClusterFormationTasks {
              * getting the short name requiring the path to already exist.
              */
             final Object esKeystoreUtil = "${-> node.binPath().resolve('elasticsearch-keystore').toString()}"
-            return configureExecTask(name, project, setup, node, esKeystoreUtil, 'create')
+            return configureExecTask(name, project, setup, node, esKeystoreUtil, 'create', '--nopass')
```

Comment:
The requirement of --nopass is a breaking change for anyone with scripts to perform setup, so I'd lean for making nopass the default in 7.x with a deprecation